### ISSUE TITLE
Extra indentation with ? operator

### ISSUE THREAD

**Issue by mathstuf** [rust-lang-nursery/rustfmt#1291]
`rustfmt` is giving me this diff after running `untry`:

```diff
-            try!(fs::create_dir_all(&gitfiledir).chain_err(|| {
-                format!("failed to create the {} submodule directory for the workarea",
-                        name)
-            }));
+            fs::create_dir_all(&gitfiledir).chain_err(|| {
+                    format!("failed to create the {} submodule directory for the workarea",
+                            name)
+                })?;
```

Why the extra indentation all of a sudden?

**Comment by nrc** [comment]
It is caused by the `?` since Rustfmt thinks this is a chain of method calls and the extra indent is necessary to indent that properly. This is a bug, and I'm pretty sure its a duplicate of another similar issue, but I can't find it right now.

**Comment by nrc** [comment]
Closed by #1502
